modules/core/src/lib/view-manager.js

```diff
@@ -77,6 +77,17 @@ export default class ViewManager {
     this._needsRedraw = this._needsRedraw || reason;
   }
 
+  updateViewStates(animationProps = {}) {
```

Comment:
Naming: How is this updating view states? There was a comment in deck, but maybe a comment above is appropriate?